docs/index.html

```diff
@@ -278,8 +279,8 @@ <h2 id="resampling">Convolution Resampling</h2>
         <p>
             From the very beginning, PIL and Pillow resampling
             performance used to be quite low
-            and comparable to ImageMagick.
-            Pillow 2.7 reverse the trend
+            compared to ImageMagick.
```

Comment:
> Pillow performance low compared to ImageMagick

This means "Pillow performance < ImageMagick performance"

I tried to say that performance of both libraries used to be very similar.